Makefile

```diff
@@ -68,6 +68,10 @@ build.arm:
 build.arm64:
 	CGO_ENABLED=0 GOOS=linux GOARCH=arm64 go build ${LDFLAGS} -o _output/bin/descheduler sigs.k8s.io/descheduler/cmd/descheduler
 
+build.ppc64le:
+        CGO_ENABLED=0 GOOS=linux GOARCH=ppc64le go build ${LDFLAGS} -o _output/bin/descheduler sigs.k8s.io/descheduler/cmd/descheduler
+
+
```

Comment:
```suggestion
```

nit: let's be consistent with the rest of the Makefile and use a single new line